### PR TITLE
Prerender manage origins

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds a section to the documentation about managing origins, in particular incorporating the Origin Settings and 404 Identifier tabs.

### CODE REVIEW

File: docs/photoniq/prerendering/manage-origins/view-origin-settings.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+- [Get origins](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins/get)
+- [Get an origin](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins-origin/get)
+
+</TabItem>
+</Tabs>
+
+## Origin Settings
+
+Dynamic Prerendering provides the following origin settings. Field names in the web console are different than those in the API responses. Refer to the API endpoint documentation for a full explanation of the API response.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/manage-origins/view-origin-settings.md]:
```suggestion
Dynamic Prerendering provides the following origin settings. Field names in the web console are different than those in the API responses. Refer to the API endpoint documentation for a full explanation.
```

File: docs/photoniq/prerendering/manage-origins/view-origin-settings.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+- **Origin URL** - Full URL of the origin.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/manage-origins/view-origin-settings.md]:
```suggestion
- **Origin URL** - URL of the origin.
```
Origin URL is just `https://www.origin.com`, for example. Maybe `Full` will give the idea of having something more than this.

File: docs/photoniq/prerendering/manage-origins/view-404-identifiers.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 30
+title: View 404 Identifiers
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+You have several options for viewing Dynamic Prerendering 404 identifiers currently set on your origins. The 404 identifier is set when an origin is created. It signals to the service that the page is a 404 and should not be rendered.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: We also render 404 pages for the bots, but what we do is set the response status code to 404.
What about this?
```suggestion
You have several options for viewing Dynamic Prerendering 404 identifiers currently set on your origins. The 404 identifier is set when an origin is created. It signals to the service that the page is rendered with a 404 status code.
```
Do you agree, @tushar-macrometa, @Luba89?

File: docs/photoniq/prerendering/manage-origins/view-404-identifiers.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- [Get an origin](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins-origin/get)
+
+</TabItem>
+</Tabs>
+
+## 404 Identifier Fields
+
+Dynamic Prerendering provides the following fields in the 404 Identifier tab. Field names in the web console are different than those in the API responses. Refer to the API endpoint documentation for a full explanation of the API response.
+
+- **Origin URL** - Full URL of the origin.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: Same suggestion here

File: docs/photoniq/prerendering/manage-origins/view-404-identifiers.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+- **Identifier String** - Text that signals the prerendering bot to ignore the page.
```
Comment (sandro-macrometa) [inline docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: Here is the same idea, we set the response status code to 404 when we find this `Identifier String` in the rendered page.

**Review by sandro-macrometa — CHANGES_REQUESTED** [review]
I left a few suggestions. Let me know what you think.

File: docs/photoniq/prerendering/manage-origins/view-404-identifiers.md (hunk 4 above)
Comment (diana-macrometa) [reply docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: Same as what? Isn't this just the origin URL?

File: docs/photoniq/prerendering/manage-origins/view-404-identifiers.md (hunk 5 above)
Comment (diana-macrometa) [reply docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: What should the exact wording here?

```
String that signals the bot to set the response code to 404.
```
?

File: docs/photoniq/prerendering/manage-origins/view-404-identifiers.md (hunk 4 above)
Comment (sandro-macrometa) [reply docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: `**Origin URL** - URL of the origin.`

Comment (diana-macrometa) [reply docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: Updated

File: docs/photoniq/prerendering/manage-origins/view-404-identifiers.md (hunk 5 above)
Comment (sandro-macrometa) [reply docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: Yes, something like:
`Text that signals the prerendering bot to set the response code to 404.`

Comment (diana-macrometa) [reply docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: Updated

File: docs/photoniq/prerendering/manage-origins/view-404-identifiers.md (hunk 3 above)
Comment (tushar-macrometa) [reply docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: I think it should be something like:  

> You have several options for viewing Dynamic Prerendering 404 identifiers currently set on your origins. The 404 identifier is set when an origin is created. It signals to the service that report a page with 404 status code where the rendered page includes the 404 identifier string.

Comment (sandro-macrometa) [reply docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: An improvement:
```
It signals to the service to report a page with a 404 status code when the
rendered page includes the 404 identifier string.
```
Or simplifying a little bit.
```
It signals to the service to report a 404 status code when the
rendered page includes the 404 identifier string.
```

Comment (diana-macrometa) [reply docs/photoniq/prerendering/manage-origins/view-404-identifiers.md]: Updated.

**Review by sandro-macrometa — APPROVED** [review]

**Pull request merged**